packages/block-library/src/image/style.scss

```diff
@@ -16,24 +16,6 @@
 		width: 100%;
 	}
 
-	// Floats get an extra wrapping <div> element, so the <figure> becomes a child.
-	.alignleft,
-	.alignright,
-	.aligncenter,
-	&.is-resized {
```

Comment:
Could the removal of this have something to do with resizing being broken?

Reply:
Mmm seems related yes

Reply:
I did test before removing this but probably missed the "large caption" use-case.

Reply:
Fixed it in the last commit

Reply:
I wonder if it would be better to generally add to the image styles, but we can explore that separately.

Reply:
Now that I think about it, I wonder if we should also restore the other selectors, since this applies also in the frontend and they could be needed there for large captions.

Reply:
Yeah, I agree we should restore

Reply:
I restored them. Do you thing there might be other front-end classes that are accidentally removed?

Reply:
I'll check but I don't expect it, I was aware of these, just missed the image ones for some reason.